docs/public-networks/reference/api/index.md

```diff
@@ -7061,6 +7063,8 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"trace_filter","params":[{"fromBl
 </TabItem>
 </Tabs>
 
+The default block range limit for `trace_filter` is 1000. You can adjust this limit using the `--rpc-max-trace-filter-range` option. 
```

Comment:
I'm not sure this needs to be mentioned here, but if it does, please move it up into the description for this method ("Returns traces matching the specified filter.") instead of after the example.